examples/addons/azure-policy/README.md

```diff
@@ -93,12 +93,25 @@ kubectl get pods -n kube-system
 | Name           | Required | Description                       | Default Value                                        |
 | -------------- | -------- | --------------------------------- | ---------------------------------------------------- |
 | name           | no       | container name                    | "gatekeeper"                                         |
-| image          | no       | image                             | "quay.io/open-policy-agent/gatekeeper:v3.0.4-beta.2" |
+| image          | no       | image                             | "mcr.microsoft.com/oss/open-policy-agent/gatekeepe:v3.1.0-beta.7" |
 | cpuRequests    | no       | cpu requests for the container    | "100m"                                               |
 | memoryRequests | no       | memory requests for the container | "256Mi"                                              |
-| cpuLimits      | no       | cpu limits for the container      | "100m"                                               |
+| cpuLimits      | no       | cpu limits for the container      | "1000m"                                              |
 | memoryLimits   | no       | memory limits for the container   | "512Mi"                                              |
 
+## Disable Azure Policy Add-on
+
+### Option 1
+
+- Update `apimodel.json` and set `azure-policy` addon `enabled` to `false`.
+- Run `aks-engine upgrade` with the updated `apimodel.json`
+- Run `kubectl delete deployments.apps -n kube-system gatekeeper-controller-manager azure-policy` to remove running deployments and pods
+
+### Option 2
+
+- SSH into AKS-Engine Kubernetes master node
+- Run `kubectl delete -f /etc/kubernetes/addons/azure-policy-deployment.yaml && sudo rm /etc/kubernetes/addons/azure-policy-deployment.yaml` to remove all resources created by addon
```

Comment:
In this option we'll also have to manually delete the deployments and apps resources, as removing the spec from the kube-addon-manager origin won't induce that deletion automatically. (Unless I'm wrong about that and you proved that the deletion does happen automatically?)

Reply:
yes, `kubectl delete -f /etc/kubernetes/addons/azure-policy-deployment.yaml` would delete the deployed resources in this case (there is `&&` in there)